pandora-client-web/src/components/wardrobe/modules/wardrobeModuleLockSlot.tsx

```diff
@@ -468,3 +512,58 @@ function PasswordInput({
 		</>
 	);
 }
+
+function TimerInput({
+	onChange,
+	timer,
+	pendingAttempt = false,
+}: {
+	onChange: (newValue: number) => void;
+	timer: Immutable<NonNullable<LockSetup['timer']>>;
+	pendingAttempt?: boolean;
+	showInvalidWarning?: boolean;
+}) {
+	const id = useId();
+
+	const [hours, setHours] = useState<number>(0);
+	const [minutes, setMinutes] = useState<number>(0);
+
+	const maxHours = useMemo(() => {
+		return timer.maxMinutes / 60;
+	}, [timer]);
+
+	const maxMinutes = useMemo(() => {
+		return timer.maxMinutes < 59 ? timer.maxMinutes : 59;
+	}, [timer]);
+
+	useEffect(() => {
+		const minutesTotal = hours * 60 + minutes;
+		onChange(minutesTotal > timer.maxMinutes ? timer.maxMinutes : minutesTotal);
+	}, [timer, onChange, hours, minutes]);
+
+	return (
+		<Row className='WardrobeInputRow'>
+			<label htmlFor={ id }>
+				Timer
+			</label>
+			<NumberInput
+				min={ 0 }
+				max={ maxHours }
+				step={ 1 }
+				value={ hours }
+				onChange={ setHours }
+				disabled={ pendingAttempt }
+			/>
+			<label>Hours</label>
```

Comment:
All labels should have unique value of `htmlFor`, matching the input they are for.
This means that optimally each `<NumberInput` should have unique id (e.g. `${id}-hours`, `${id}-mintues`) and the same should be set on its label - label describes what is the name of particular input, mainly important for assistive technologies (e.g. screen readers).